ctx.go

```diff
@@ -661,7 +661,7 @@ func (c *Ctx) GetRespHeaders() map[string]string {
 func (c *Ctx) Hostname() string {
 	if c.IsProxyTrusted() {
 		if host := c.Get(HeaderXForwardedHost); len(host) > 0 {
-			return host
+			return strings.Split(host, ",")[0]
```

Comment:
I think you can use strings.Index instead. Split causes extra alloc

Reply:
Thanks for suggestion, applied already 😄